config/runtimes/kserve-huggingfaceserver-multinode.yaml

```diff
@@ -0,0 +1,210 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ServingRuntime
+metadata:
+  name: kserve-huggingfaceserver-multinode
```

Comment:
I wonder if we should mention ray/vllm here

Reply:
I just add `note` in [the doc ](https://github.com/kserve/website/pull/402/files#diff-5cc839fa31bb0809065051dca6ba9c17747ce13bf3707a4bec5cfb0027c980f2R18) but if we need to specify it in the runtime title, I am ok with it. @yuzisun wdyt?